src/main/java/core/basesyntax/service/TypeOfOperation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public enum TypeOfOperation {
+    s, b, r
```

Comment:
Enums values should be capital. And considering this - I think no pint to have Enums...